plugins/modules/apt_rpm.py

```diff
@@ -37,7 +37,16 @@
   state:
     description:
       - Indicates the desired package state.
-    choices: [ absent, present, installed, removed ]
+      - Please note that V(present) and V(installed) are equivalent to V(latest) right now.
+        This will change in the future.
```

Comment:
```suggestion
        This will change in the future. To simply ensure that a package is installed, without upgrading
        it, use the V(present_not_latest) state.
```